test/fileurlreader.cpp

```diff
@@ -102,21 +103,26 @@ TEST_CASE("Preserves URLs as-is", "[FileUrlReader][issue926]")
 		R"_(exec:curl --silent https://feeds.metaebene.me/raumzeit/m4a  | sed 's#\(</guid>\|</id>\)#-M4A&#')_");
 }
 
-TEST_CASE("URL reader returns error message if file cannot be opened",
+TEST_CASE("URL reader returns error structure if file cannot be opened",
 	"[FileUrlReader]")
 {
 	const std::string testDataPath("data/test-urls.txt");
 
 	TestHelpers::TempFile urlsFile;
 	FileUrlReader u(urlsFile.get_path());
 
-	SECTION("reload() returns error message if file does not exist") {
+	SECTION("reload() returns error structure if file does not exist") {
 		const auto error_message = u.reload();
 		REQUIRE(error_message.has_value());
-
+		/**
 		SECTION("the error message contains the filename") {
-			INFO("error_message: " + error_message.value());
-			REQUIRE(error_message.value().find(urlsFile.get_path()) != std::string::npos);
+			INFO("error_message: " + error_message.value().message);
+			REQUIRE(error_message.value().message.find(urlsFile.get_path()) != std::string::npos);
+		}
+		*/
+		SECTION("the error structure contains error message and kind of an error") {
+			INFO("error_message: " + error_message.value().message);
```

Comment:
Please change this to a `REQUIRE` that checks that the message is non-empty; it's better than logging because it's automated.